src/actions/index.js

```diff
@@ -0,0 +1,15 @@
+import huoa from '../api/huoa';
```

Comment:
lets change this to be a bit more agnostic something more like 
```
import api from '../base/api';
```

Reply:
done > src/apis/content.js